database/core_templates/snapshot.json

```diff
@@ -7503,7 +7534,90 @@
    "title": "Automatic"
   }
  ],
- "activityLog": [],
+ "activityLog": [
+  {
+   "applicationId": 18,
+   "details": {
+    "outcome": "PENDING"
+   },
+   "id": 2,
+   "recordId": 18,
+   "table": "application",
+   "timestamp": "2022-08-11T10:13:08.263223+12:00",
+   "type": "OUTCOME",
+   "value": "PENDING"
+  },
+  {
+   "applicationId": 18,
+   "details": {
+    "stage": {
+     "name": "Check Username",
+     "colour": "#1E14DB",
+     "number": 1
+    }
+   },
+   "id": 3,
+   "recordId": 17,
+   "table": "application_stage_history",
+   "timestamp": "2022-08-11T10:13:08.437572+12:00",
+   "type": "STAGE",
+   "value": "Check Username"
+  },
+  {
+   "applicationId": 18,
+   "details": {
+    "status": "DRAFT",
+    "prevStatus": null
+   },
+   "id": 4,
+   "recordId": 17,
+   "table": "application_status_history",
+   "timestamp": "2022-08-11T10:13:08.440729+12:00",
+   "type": "STATUS",
+   "value": "DRAFT"
+  },
+  {
+   "applicationId": 19,
+   "details": {
+    "outcome": "PENDING"
+   },
+   "id": 5,
+   "recordId": 19,
+   "table": "application",
+   "timestamp": "2022-08-11T10:15:21.574247+12:00",
+   "type": "OUTCOME",
+   "value": "PENDING"
+  },
+  {
+   "applicationId": 19,
+   "details": {
+    "stage": {
+     "name": "Check Username",
+     "colour": "#1E14DB",
+     "number": 1
+    }
+   },
+   "id": 6,
+   "recordId": 18,
+   "table": "application_stage_history",
+   "timestamp": "2022-08-11T10:15:21.714024+12:00",
+   "type": "STAGE",
+   "value": "Check Username"
+  },
+  {
+   "applicationId": 19,
+   "details": {
+    "status": "DRAFT",
+    "prevStatus": null
+   },
+   "id": 7,
+   "recordId": 18,
+   "table": "application_status_history",
+   "timestamp": "2022-08-11T10:15:21.717953+12:00",
+   "type": "STATUS",
+   "value": "DRAFT"
+  }
```

Comment:
I think we can delete this stuff. Just put it back to an `"activityLog": []` like it was before. (Just manually edit this file)

Reply:
Nice - I've removed those to '"activityLog": [ ]'.

I see maybe that it isn't viewable in github because it thinks of it as a massive delete + massive addition

Reply:
It's because the auto-formatter changed the whole file to match Prettier formatting. 

There is an option so "Save without formatting" that is quite useful in these cases.

Reply:
I see, good as gold, I'll be aware of that in future